ethereum/rpc/types/utils.go

```diff
@@ -206,23 +206,16 @@ func FormatBlock(
 	}
 }
 
-func DecodeTx(clientCtx client.Context, txBz tmtypes.Tx) (sdk.Tx, uint64) {
-	var gasUsed uint64
+//DecodeTx decode tendermint tx to a cosmos tx
+func DecodeTx(clientCtx client.Context, txBz tmtypes.Tx) sdk.Tx {
 	txDecoder := clientCtx.TxConfig.TxDecoder()
 
 	tx, err := txDecoder(txBz)
 	if err != nil {
-		return nil, 0
+		return nil
 	}
 
-	switch tx := tx.(type) {
-	case *evmtypes.MsgEthereumTx:
-		gasUsed = tx.GetGas() // NOTE: this doesn't include the gas refunded
-	case sdk.FeeTx:
-		gasUsed = tx.GetGas()
-	}
-
-	return tx, gasUsed
+	return tx
 }
```

Comment:
I guess we can remove this now that we are not returning the gas used and just use the decoder from the `TxConfig`

Reply:
ok removed